src/main/java/com/laserfiche/api/client/apiserver/TokenClientImpl.java

```diff
@@ -1,44 +1,23 @@
 package com.laserfiche.api.client.apiserver;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.DeserializationFeature;
-import com.fasterxml.jackson.databind.MapperFeature;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.json.JsonMapper;
-import com.fasterxml.jackson.databind.module.SimpleModule;
-import com.laserfiche.api.client.deserialization.OffsetDateTimeDeserializer;
-import com.laserfiche.api.client.model.*;
-import com.laserfiche.api.client.deserialization.TokenClientObjectMapper;
-import kong.unirest.Header;
-import kong.unirest.HttpResponse;
+import com.laserfiche.api.client.model.ApiException;
+import com.laserfiche.api.client.model.CreateConnectionRequest;
+import com.laserfiche.api.client.model.ProblemDetails;
+import com.laserfiche.api.client.model.SessionKeyInfo;
+import com.laserfiche.api.client.oauth.OAuthClient;
 import kong.unirest.Unirest;
 import kong.unirest.json.JSONObject;
-import org.threeten.bp.OffsetDateTime;
 
-import java.util.HashMap;
 import java.util.Map;
 import java.util.concurrent.CompletableFuture;
-import java.util.stream.Collectors;
 
-public class TokenClientImpl implements TokenClient {
+public class TokenClientImpl extends OAuthClient implements TokenClient {
     private String baseUrl;
```

Comment:
It seems we will never change this variable once assigned. We could mark this as final to show this intention.

Reply:
ok renamed and capatalized everything to follow the java naming convention for constants